sui/utils.js

```diff
@@ -48,7 +49,21 @@ const loadSuiConfig = (env) => {
     return config;
 };
 
+const deployPackage = async (packageName, client, keypair) => {
+    const compileDir = `${__dirname}/move`;
+    copyMovePackage(packageName, null, compileDir);
+    const builder = new TxBuilder(client);
+    await builder.publishPackageAndTransferCap(packageName, keypair.toSuiAddress(), compileDir);
```

Comment:
```suggestion
    await builder.publishPackageAndTransferCap(packageName, options.owner || keypair.toSuiAddress(), compileDir);
```
allow specifying an owner for the cap instead of just using the deployer

Reply:
done